index.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>World-Congress on Quantum Physics</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet">
+    <link rel="stylesheet" href="style.css" />
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+
+  </head>
+
+
+  <body>
+    <nav class="navbar navbar-expand-lg position-absolute 
+    container-fluid d-flex justify-content-between align-items-center nav-color p-3 ps-lg-5">
+
+          <a class="navbar-brand logo" href="index.html"><img src="images/logo.png" alt='quantum' width="30" height="30"></a>
+          <button class="navbar-toggler hamburger">
+            <span class="navbar-toggler-icon"></span>
+          </button>
+          <div class="nav-menu collapse navbar-collapse justify-content-end">
+            <ul class="navbar-nav menu">
+              <li class="nav-item ps-lg-5">
+                <a class="nav-link" href="about.html">About</a>
+              </li>
+              <li class="nav-item ps-lg-5">
+                <a class="nav-link" href="#">Program</a>
+              </li>
+              <li class="nav-item ps-lg-5">
+                <a class="nav-link" href="#">Join</a>
+              </li>
+              <li class="nav-item ps-lg-5">
+                <a class="nav-link" href="#">Sponsor</a>
+              </li>
+              <li class="nav-item ps-lg-5">
+                <a class="nav-link" href="#">News</a>
+              </li>
+              <li class="nav-item ps-lg-5">
+                <a class="nav-link" href="#">CC Campaign</a>
+              </li>
+            </ul>
+          </div>
+      
+      </nav>
```

Comment:
- [ ] Please wrap your `<navbar>` bar tag with a `<Header>` tag because it will be better semantically that way.